### ISSUE TITLE
Add a --fix option for no-useless-constructor

### ISSUE THREAD

**Issue by not-an-aardvark** [eslint/eslint#6995]
I'm using Node v6.4.0 and ESLint v3.4.0.

It would be useful to have an autofix option for the `no-useless-constructor` rule. By [definition](http://eslint.org/docs/rules/no-useless-constructor#rule-details), all constructors reported by this rule can be safely removed without changing how the class works, so the fixer would just remove them.

``` js
class Foo {
  constructor(a, b) {
  }
}

class Bar extends Foo {
  constructor(c, d) {
    super(c, d);
  }
}

// gets fixed to:

class Foo {

}

class Bar extends Foo {

}
```

I would be willing to add this if the issue is accepted.

**Comment by platinumazure** [comment]
Personally I'm not sure about this one- I could see myself writing a placeholder constructor, intending to fill it in later, only for the linter to wipe it out. Could be a frustrating user experience.

Let's see what the rest of the team thinks.

**Comment by mysticatea** [comment]
Sounds good to me.
If a developer wants to write empty constructors, they will disable this rule.

...But if an empty constructor has (JSDoc) comment, I think it should not remove the constructor automatically. (maybe it should not warn such constructors?)

**Comment by not-an-aardvark** [comment]
I'm closing this issue since we didn't reach consensus for a long time, and there are other issues that should have a higher priority.
